install/centos7_rpm_install.md

```diff
@@ -0,0 +1,125 @@
+# 两步完成 APISIX 在 CentOS 7 安装
```

Comment:
I think the use of `APISIX` in the article is more concise. Because there are multiple occurrences of `APISIX` in the article, in order to keep the name consistent, we also need to use `Apache APISIX` to replace `APISIX`, so it seems not concise enough.